ir_actions_server_security_adj/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module gives read access for server actions to the internal users.
```

Comment:
Do you know what the background is that internal users need read access to?  (I'll check it too.)

Reply:
I understand the background, We need this module to hide the action of ir.action.server from non-internal users.
Could you please add this context to readme??

Reply:
I am wonder non-internal users can see server actions. For portal users, I think they can't see any server action in portal page. Is there any module that allow to see server actions for non-internal users?

Reply:
Yes. The odoo_marketplace module functionality allows non-internal users to enter the backend.